client/src/components/App.tsx

```diff
@@ -98,11 +98,12 @@ const App = (): ReactElement => {
             backgroundSize: 'cover',
             backgroundRepeat: 'no-repeat',
-            minHeight: window.screen.height,
+            minHeight: (window.screen.height * .889),
+            width: window.screen.width
           }}
         >
           <UserProvider>
-            {['/dashboard', '/', '/survey'].includes(location.pathname) ? <></> : <Nav />}\
+            {['/', '/survey'].includes(location.pathname) ? <></> : <Nav />}\
```

Comment:
is the '\' intentional at the end?

Reply:
I think github removed the dash from your comment, I was so confused with what you were trying to say lol